docs.package.json

```diff
@@ -95,5 +100,29 @@
     "source": "docs/images/*.png",
     "target": "docs/04-operating-scs/components",
     "label": "automated-pentesting/images"
+  },
+  {
+    "repo": "SovereignCloudStack/csctl",
+    "source": "docs/*.md",
+    "target": "docs/03-container/components/cluster-stacks/components",
+    "label": "csctl"
+  },
+  {
+    "repo": "SovereignCloudStack/cluster-stacks",
+    "source": "docs/overview.md",
+    "target": "docs/03-container/components/cluster-stacks/components",
+    "label": "cluster-stacks"
+  },
+  {
+    "repo": "SovereignCloudStack/cluster-stack-operator",
+    "source": ["docs/architecture/overview.md"],
+    "target": "docs/03-container/components/cluster-stacks/components",
+    "label": "cluster-stack-operator"
+  },
+  {
+    "repo": "SovereignCloudStack/cluster-stack-operator",
+    "source": ["docs/pics/syself-cluster-stacks-web.png"],
+    "target": "docs/03-container/components/cluster-stacks/components",
+    "label": "pics"
```

Comment:
Hi @DEiselt, I had to add the CSO twice because the `overview.md` includes a picture which is in the relative path `../pics` which is fine, but obviously `getDocs.js` doesn't care about that and puts both files in one dir, which leaded to this error:
```
Error: MDX compilation failed for file "/data/docs/03-container/components/cluster-stacks/components/cluster-stack-operator/overview.md"
Cause: Image docs/03-container/components/cluster-stacks/components/pics/syself-cluster-stacks-web.png used in docs/03-container/components/cluster-stacks/components/cluster-stack-operator/overview.md not found.
Details:
Error: Image docs/03-container/components/cluster-stacks/components/pics/syself-cluster-stacks-web.png used in docs/03-container/components/cluster-stacks/components/cluster-stack-operator/overview.md not found.
```
With this workaround it does.

Reply:
Also CSO has many chars in all the docs, that's why I only included this two files first to have the card in the Docs structure

Reply:
Created https://github.com/SovereignCloudStack/cluster-stack-operator/pull/246 to fix linter issues and restructure how CSO docs are pulled in. In my test this also worked with the image and other relative links.

We should be able to resolve this conversation.